redisson/src/test/java/org/redisson/RedisRunner.java

```diff
@@ -871,7 +870,7 @@ private void makeRandomDefaultDir() {
             f.mkdirs();
             this.defaultDir = f.getAbsolutePath();
             if (RedissonRuntimeEnvironment.isWindows) {
-                defaultDir = defaultDir.replace("\\", "\\\\");
+                defaultDir = "\"" + defaultDir.replace("\\", "\\\\") + "\"";
```

Comment:
On macos 10.14 with JDK1.8.0_162 or JDK9, when running the unit test inside IDE, it will fail with flowing message(I've try redis 3.2, 4.0.6, 4.0.9, 4.0.11 and 5.0.0):

```
REDIS RUNNER: Making directory /var/folders/fv/qgq4zr4n08d3kfv27g_75pdc0000gp/T/18801a06-93e9-49e9-b54a-0ff90b6ee185
REDIS LAUNCH OPTIONS: [/usr/local/opt/redis@3.2/bin/redis-server, --port, 6379, --dir, "/var/folders/fv/qgq4zr4n08d3kfv27g_75pdc0000gp/T/18801a06-93e9-49e9-b54a-0ff90b6ee185", --notify-keyspace-events, Exg]
REDIS PROCESS: 24496:C 12 Nov 10:03:38.597 # Can't chdir to '"/var/folders/fv/qgq4zr4n08d3kfv27g_75pdc0000gp/T/18801a06-93e9-49e9-b54a-0ff90b6ee185"': No such file or directory
```

When I remove the wrapped `""`, it start the redis server successfully. So I move it to windows environment. If this is incidental, please let me know and I will change it back.

By the way, I cannot test this on Windows and Linux, Sorry for this.

Reply:
Revert back to the master version